src/RequestForm.js

```diff
@@ -616,9 +654,12 @@ class RequestForm extends React.Component {
           <Pane
             defaultWidth="100%"
             height="100%"
-            firstMenu={addRequestFirstMenu}
-            lastMenu={isEditForm ? editRequestLastMenu : addRequestLastMenu}
-            actionMenu={renderActionMenu}
+            firstMenu={this.renderAddRequestFirstMenu()}
+            lastMenu={isEditForm
```

Comment:
I think it will be a little bit cleaner to move this condition to `renderLastMenu` method